setup.py

```diff
@@ -155,9 +155,14 @@
 ]
 extras_require = {
     ':sys_platform != "win32"': ['terminado>=0.3.3'],
-    'doc': ['Sphinx>=1.1'],
+    'doc': [
+        'Sphinx>=1.1',
```

Comment:
Sphinx should really be 1.3.6 or above to properly support Markdown and RST as sources and avoid some errors found on kernel gateway project.